source/test/Config.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Klemens D. Morgenstern
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include "Config.hpp"
+#include <mrdox/Error.hpp>
+#include <llvm/Support/FileSystem.h>
+#include <llvm/Support/Path.h>
+#include <llvm/Support/YAMLParser.h>
+#include <llvm/Support/YAMLTraits.h>
+
+
+template<>
+struct llvm::yaml::MappingTraits<
+    clang::mrdox::TestConfig>
+{
+  static void mapping(IO &io,
+                      clang::mrdox::TestConfig& f)
```

Comment:
It would be best if we did not have two files with the same name anywhere, we already have Config.cpp in api